<?xml version="1.0"?>
<!--肢体控制-->
<launch>
    <include file="$(find hiwonder_peripherals)/launch/depth_cam.launch"/>
    <include file="$(find hiwonder_controller)/launch/hiwonder_controller.launch"/>
    <include file="$(find hiwonder_servo_controllers)/launch/start.launch"/>

    <node name="init_pose" pkg="hiwonder_slam" type="init_pose.py" output="screen">
        <param name="horizontal" value="true"/>
    </node>

    <node pkg="hiwonder_example" type="body_control.py" name="body_control" output="screen"/>
</launch>
